src/main/java/edu/harvard/iq/dataverse/search/SearchFields.java

```diff
@@ -220,6 +220,8 @@
      * i.e. "Unpublished", "Draft" (multivalued)
      */
     public static final String PUBLICATION_STATUS = "publicationStatus";
+    
+    public static final String EXTERNAL_STATUS = "externalStatus";
```

Comment:
Can you facet on this new externalStatus Solr field?

Reply:
You can't. There is the superuser API call to get the CSV table of all draft datasets and their current labels. I hope that faceting to select draft-only within a given sub-collection will make things usable without having to also facet here but it could be added (presumably listing the labels from all sets if you're looking at collections that use different sets?)